dat/generated_tables.py

```diff
@@ -0,0 +1,73 @@
+from typing import Sequence
+
+from dat.model.row_collections import RowCollection
+from dat.model.table import GeneratedReferenceTable, ReferenceTable
+
+reference_table_1 = GeneratedReferenceTable(
+    table_name='reference_table_1',
+    table_description='My first table',
+    column_names=['letter', 'number', 'a_float'],
+    partition_keys=['letter'],
+    reader_protocol_version=2,
+    writer_protocol_version=2,
+    row_collections=[
+        RowCollection(
+            write_mode='overwrite',
+            data=[
+                ('a', 1, 1.1),
```

Comment:
It's a good point. A simple solution could be to add a field to the reference table model to define the schema, and having this schema defined in terms of PySpark types.  However, if there is no 1-to-1 mapping between PySpark types and Delta lake types, it will not be a complete solution. What do you think?

Reply:
Yeah for the generated it does sounds like having it written in PySpark makes sense, since the schema should be able to be passed to `spark.createDataFrame()` and get it to interpret the types correctly. 👍